app/multitenant/aws_collector.go

```diff
@@ -338,45 +337,55 @@ func (c *awsCollector) Report(ctx context.Context) (report.Report, error) {
 	return c.merger.Merge(reports), nil
 }
 
+// calculateDynamoKeys generates the row & column keys for Dynamo.
+func calculateDynamoKeys(userid string, now time.Time) (string, string) {
+	rowKey := fmt.Sprintf("%s-%s", userid, strconv.FormatInt(now.UnixNano()/time.Hour.Nanoseconds(), 10))
+	colKey := strconv.FormatInt(now.UnixNano(), 10)
+	return rowKey, colKey
+}
+
+// calculateReportKey determines the key we should use for a report.
+func calculateReportKey(rowKey, colKey string) (string, error) {
```

Comment:
Because everywhere else, we call it a "report key". Further, the key is used for memcache as well as S3. 

If still you think "s3 key" would be better after this explanation, then let me know & I'll change it.